Tensor construction: combine Resize+mutable_data - 1/4 (#13853)

jerryzh168 opened the pull request:
Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/13853

Codemod generated with clangr shard mode, 25 files per diff,
motivation: https://github.com/pytorch/pytorch/pull/12407

Differential Revision: D13054770

smessmer reviewed (approved):
see comments in phabricator

jerryzh168 committed:
Tensor construction: combine Resize+mutable_data - 1/4 (#13942)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/13942

Codemod generated with clangr shard mode, 25 files per diff,
motivation: https://github.com/pytorch/pytorch/pull/12407

Reviewed By: smessmer

Differential Revision: D13054770

fbshipit-source-id: be2d76996f78f60ba875234aa5eb7a7e081ab833
